src/Native/Unix/System.Native/pal_networkstatistics.c

```diff
@@ -38,6 +38,12 @@
 #include <netinet/icmp6.h>
 #include <netinet/icmp_var.h>
 
+
+enum {
+    InteropInterfaceUp = 0x1,
+    InteropSupportsMulticast = 0x2
+};
```

Comment:
Shouldn't this be in pal_networkstatistics.h?  Seems like it's effectively part of TcpGlobalStatistics's definition.